File: LiveArchiveOJ/4215_Feynman/Main.java
/*
Alejandro Santos - 2012
https://github.com/alejolp/ACM-ICPC-Stuff
*/

import java.io.IOException;
import java.util.Scanner;

/* IMPORTANTE: La clase debe llamarse Main */

public class Main {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
 
        while (true) {
            int x = s.nextInt();
            if (x == 0) break;
            System.out.println((x * (x + 1) * (2 * x + 1)) / 6);
        }
    }
}

File: LiveArchiveOJ/4215_Feynman/main.c
/*
Alejandro Santos - 2012
https://github.com/alejolp/ACM-ICPC-Stuff
*/


#include <stdio.h>

int main(int argc, char** argv) {
    int n;
    while (scanf("%d", &n) && n) {
        /* sumatoria de k=1..n de k^2 */
        printf("%d\n", (n * (n + 1) * (2 * n + 1)) / 6);
    }
    return 0;
}

